src/app/components/email-validation/index.html

```diff
@@ -3,111 +3,111 @@
-                    <li><a routerLink="/components/url-validation">URL validation</a></li>
-                </ul>
-            <h3>Guidelines</h3>
-                <ul>
-                    <li><a routerLink="/design/guidelines/error-handling">Error handling</a></li>
-                    <li><a routerLink="/design/guidelines/form-design">Form design</a></li>
-                </ul>
-	   </sky-demo-page-content>
+      <sky-demo-page-content sectionHeading="Related information">
+        <h3>Components</h3>
+          <ul>
+            <li><a routerLink="/components/form">Form</a></li>
+            <li><a routerLink="/components/status-indicator">Status indicator</a></li>
+            <li><a routerLink="/components/url-validation">URL validation</a></li>
+          </ul>
+        <h3>Guidelines</h3>
+          <ul>
+            <li><a routerLink="/design/guidelines/error-handling">Error handling</a></li>
+            <li><a routerLink="/design/guidelines/form-design">Form design</a></li>
+          </ul>
+	  </sky-demo-page-content>
     </sky-tab>
     
     <sky-tab tabHeading="Development">
-    
-        <sky-demo-page-module-info
+      <sky-demo-page-module-info
 		skyuxModule="SkyEmailValidationModule"
 		packageName="@skyux/validation"
         githubRepo="skyux-validation"
-        >
-	   </sky-demo-page-module-info>
+      >
+      </sky-demo-page-module-info>
         
-        <sky-demo-page-properties sectionHeading="Email validation input directive properties">
-        <sky-demo-page-property
-          propertyName="skyEmailValidation"
-        >
-            Creates an input that validates email addresses. Place this attribute on an <sky-code>input</sky-code> element. If users enter values that are not valid email addresses, an error message appears. The directive uses <sky-code>NgModel</sky-code> to bind data.
-        </sky-demo-page-property>
-        </sky-demo-page-properties>
+    <sky-demo-page-properties sectionHeading="Email validation input directive properties">
+      <sky-demo-page-property
+        propertyName="skyEmailValidation"
+      >
+        Creates an input that validates email addresses. Place this attribute on an <sky-code>input</sky-code> element. If users enter values that are not valid email addresses, an error message appears. The directive uses <sky-code>NgModel</sky-code> to bind data.
+      </sky-demo-page-property>
+    </sky-demo-page-properties>
+
+    <sky-demo-page-properties sectionHeading="Email validator function properties">
+      <sky-demo-page-property propertyName="SkyValidators.email">
+        Creates an input that validates email addresses in reactive forms. Add this validator directly to the form control model in the component class. If users enter values that are not valid email addresses, an error message appears. Since this is a sync validator, it returns a set of validation errors or <sky-code>undefined</sky-code> immediately when users enter values.
```

Comment:
This doesn't really `Create(s) an input`. I would probably just make this sentence `Validates email addresses in reactive forms`. These functions add the ability to validate emails no matter how they come in so I don't think we should limit the language to just inputs.

Reply:
I also don't know that I would say that `an error message appears` as that would be on the consumer. I would say `an error is thrown`.

Reply:
Fair points - this is a bit too specific to intended form use-cases and not to the overall functionality of the ValidatorFns themselves.  Let me see if I can reword and repush.

Reply:
I tried to update in the branch locally then push changes up but I get 403 denied.  Originally I had this forked and PR'd but I think @blackbaud-johnly moved this to a feature branch to pass some checks.  Let me know if it's easier for someone else to update who has access to the feature or if I need to do this a different way.

Reply:
Thanks for trying to update. I talked to Trevor this morning, and I will make the updates. I've just been focused on something else and haven't gotten to it yet. Thanks!

Reply:
I just updated the content address this; Instead of `an error is thrown`, I opted for `the validator throws an error` to use active voice.